README.md

```diff
@@ -11,12 +11,12 @@ TorchANI is a pytorch implementation of ANI. It is currently under alpha release
 TorchANI requires the latest preview version of PyTorch. You can install PyTorch by
 
 ```bash
-conda install pytorch-nightly -c pytorch
+conda install pytorch-nightly cuda92 -c pytorch
```

Comment:
Did you try it? I don't think this works. Also, it is not a good idea to specify to a specific cuda version here in TorchANI, because TorchANI runs on all cuda versions.